ci/deploy.sh

```diff
@@ -122,7 +122,15 @@ build_pyfory() {
   echo "Install pyfory"
   # Fix strange installed deps not found
   pip install setuptools -U
-  bazel build //:cp_fory_so
+
+  # Detect host architecture and only pass x86_64 config when appropriate
+  ARCH=$(uname -m)
+  if [[ "$ARCH" == "x86_64" || "$ARCH" == "amd64" ]]; then
+    bazel build --config=x86_64 //:cp_fory_so
```

Comment:
It would be great if we can keep all build as `bazel build //:cp_fory_so`, this will make new contributor more easy to build project

Reply:
How would you like to automatically apply the right build flags from .bazelrc? We can't leave the x86 flags enabled in ARM runs.

Reply:
”automatically apply the right build flags from .bazelrc” looks good to me

Reply:
Sure. How is this done?